aya/src/programs/tc.rs

```diff
@@ -228,6 +228,33 @@ define_link_wrapper!(
     TcLinkId
 );
 
+impl SchedClassifierLink {
+    /// Creates a new SchedClassifierLink instance
```

Comment:
Could you add an example of using the `new()` method in this docstring? I think it's not obvious for users how they could use it together with `tc_options()`.

Reply:
Thanks for asking :-).  It triggered me to think it through again and I made a couple of minor changes.  I also added an example.  Please let me know if it looks okay.